docs/cudf/source/user_guide/config.md

```diff
@@ -0,0 +1,8 @@
+# Options
+
+CUDF options are a set of key value pairs stored in a global dictionary.
```

Comment:
```suggestion
cuDF options are a set of key-value pairs stored in a global dictionary.
```
Is this description useful? Compare to what [pandas writes](https://pandas.pydata.org/docs/user_guide/options.html):
> pandas has an options API [to] configure and customize global behavior related to [DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html#pandas.DataFrame) display, data behavior and more.

Reply:
Makes sense, since this is user facing guide.